src/controllers/stallController.ts

```diff
@@ -41,6 +41,9 @@ function getStallInclude(): Includeable[] {
     {
       association: Stall.associations.Categories,
     },
+    {
```

Comment:
Sending this back is quite nasty. If a Stall has 5 `Category`, that is 5 rows of `CategoryStall` that the frontend does not need.